### PR TITLE
Support for running notebooks on Binder

### PR DESCRIPTION
**Pull request by frreiss**
This PR adds the necessary configuration files and links to run our example notebooks on Binder. 

I've also added links to the README.md file. For now, the links point to the source branch for this PR. After these changes are merged, I'll update the links in the README.md in the master branch to point to master.

### CODE REVIEW

**Comment by frreiss** [comment]
See the README.md at https://github.com/frreiss/tep-fred/tree/branch-binder for Binder links.

File: README.md [hunk 1]
```diff
@@ -83,11 +95,7 @@ API documentation can be found at [https://readthedocs.org/projects/text-extensi
   cover complex end-to-end NLP use cases (work in progress).
 
 
-## Instructions to run a demo notebook
-1. Check out a copy of this repository
-1. (optional) Use the script `env.sh` to set up an Anaconda environment for running the code in this repository.
-1. Type `jupyter lab` from the root of your local source tree to start a [JupyterLab](https://jupyterlab.readthedocs.io/en/stable/) environment.
-1. Navigate to the `notebooks` directory and choose any of the notebooks there
+## Instructions to run a demo notebook locally
```
Comment (BryanCutler) [inline README.md:98]: This is covered above, did you mean to remove this line?

**Review by BryanCutler — APPROVED** [review]
This will be great to have! I tried out your link, it took a long long time to load, but then it seemed no packages could be found? Not sure if it's just my problem, but maybe best to merge and then link to the main repo and fix any issues there.

File: README.md (hunk 1 above)
Comment (frreiss) [reply README.md]: Oops!  Fixed now.

**Comment by frreiss** [comment]
Merging to facilitate additional testing.

**Pull request merged**